src/utilities/db/Birthday.ts

```diff
@@ -0,0 +1,65 @@
+import { Utility } from '@sapphire/plugin-utilities-store';
+import { extractDayAndMonth } from '../../helpers/utils/date';
+import type { Guild, User } from 'discord.js';
+
+export class Birthday extends Utility {
+	private prisma = this.container.prisma;
+
+	public constructor(context: Utility.Context, options: Utility.Options) {
+		super(context, {
+			...options,
+			name: 'birthday',
+		});
+	}
+	public get = {
+		BirthdaysByDate: (date: string) => this.prisma.birthday.findMany({ where: { birthday: { contains: extractDayAndMonth(date) } } }),
+		BirthdayByDateAndTimezone: (date: string, timezone: number) => this.prisma.birthday.findMany({ where: { birthday: { contains: extractDayAndMonth(date) }, guild: { timezone } } }),
+		BirthdaysByGuildID: (guildID: string) => this.prisma.birthday.findMany({ where: { guild_id: guildID } }),
+		BirthdayByUserAndGuild: (guildID: string, userID: string) => this.prisma.birthday.findUnique({
+			where: { user_id_guild_id: { guild_id: guildID, user_id: userID } },
+		}),
+		BirthdaysNotDisabled: (guildID: string) => this.prisma.birthday.findMany({ where: { guild_id: guildID, disabled: false } }),
+	};
+
+	public update = {
+		BirthdayDisabled: (guildID: string, userID: string, disabled: boolean) => this.prisma.birthday.update({
+			where: { user_id_guild_id: { guild_id: guildID, user_id: userID } }, data: { disabled },
+		}),
+		BirthdayByUserAndGuild: (guildID: string, userID: string, birthday: string) => this.prisma.birthday.update({
+			where: { user_id_guild_id: { guild_id: guildID, user_id: userID } }, data: { birthday },
+		}),
+	};
+
+	public delete = {
+		GuildByID: (guildID: string) => this.prisma.guild.delete({ where: { guild_id: guildID } }),
+		ByDisabledGuilds: () => this.prisma.guild.deleteMany({ where: { disabled: true } }),
+		ByLastUpdateDisable: (date: Date) => this.prisma.guild.deleteMany({ where: { last_updated: date.toISOString(), disabled: true } }),
```

Comment:
Does this delete the entrys, that have the `last_updated` value longer ago then `date.toISOString`?